node/src/command.rs

```diff
@@ -39,9 +39,9 @@ fn load_spec(
 	para_id: ParaId,
 ) -> std::result::Result<Box<dyn sc_service::ChainSpec>, String> {
 	Ok(match id {
-		"dev" => Box::new(chain_spec::development_config(para_id)),
+		"canvas-dev" => Box::new(chain_spec::development_config(para_id)),
```

Comment:
```suggestion
		"dev | canvas-dev" => Box::new(chain_spec::development_config(para_id)),
```

I don't want to lose the ability to run with `--dev` (which I believe is checked here)